pkg/kosmosctl/manifest/manifest_daemonsets.go

```diff
@@ -19,35 +19,35 @@ spec:
       labels:
         app: clusterlink-floater
     spec:
+      containers:
+        - name: floater
+          image: {{ .ImageRepository }}/clusterlink-floater:v{{ .Version }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - clusterlink-floater
+          securityContext:
+            privileged: true
```

Comment:
`privileged: true`  should be very careful, In some security testing scenarios, the tests may not pass. You could config capacities ranther than `privileged: true`

Reply:
This is correct, but before I modify it, I need to conduct some tests, because the floater does need to perform some privileged operations to ensure that the network connectivity verification is successful